### ISSUE TITLE
Not working with today's RELEASE of OS X 10.10.4 (14E46)

### ISSUE THREAD

**Issue by Fofer** [hetima/SafariStand#31]
Just updated to release of OS X 10.10.4, and the latest version of SafariStand (8.0.214) doesn't work. :(

**Comment by levifig** [comment]
:+1:

**Comment by sambuev** [comment]
yes, doesn't work :( please give me tutorials how to fix it. Everything is installed in right folder, but it dosn't show up in menu

**Comment by hetima** [comment]
[How to transition from EasySIMBL to SIMBL](https://gist.github.com/hetima/51b71091aedf622f61f7)

**Comment by Fofer** [comment]
Thanks so much, I have now transitioned (back) to SIMBL from EasySIMBL and my beloved SafariStand is back again.  Some of my other SIMBL plugins seem to be dead but none are as precious to me as SafariStand.  Is it true that all SIMBL plugins including SafariStand will be no longer possible in OS X 10.11 El Capitan?  Is there any other strategy or workaround for SafariStand to live on in OS X 10.11?

**Comment by ylluminate** [comment]
Useful for anyone building this from the shell quickly to get it working:

• Check out the source and from the folder it's downloaded to via `git`:
• `xcodebuild -list -project SafariStand.xcodeproj` # Get your build options
• `xcodebuild -scheme SafariStand -configuration Release build SYMROOT="."` # Builds to `pwd`/Release
• Stick the `SafariStand.bundle` into the appropriate SIMBL plugin folder.

**Comment by ylluminate** [comment]
@hetima are you sticking this into `/Library/Application\ Support/SIMBL/Plugins/` or `~/Library/Application\ Support/SIMBL/Plugins`?  Further, does it need `root` ownership under the `/Library/` path or can it be owned by `username:staff`?

**Comment by hetima** [comment]
I use under `~/Library` on 10.10.4 as it is.
Under `/Library` seems to work regardless of ownership.

I don't test on 10.11 yet.

**Comment by ylluminate** [comment]
Interesting to know @hetima, thanks.  I am hitting an odd wall on 10.10.4 with the Safari 9 preview build.  Not getting Stand to load it seems.  Have you seen any problems or heard of problems in such a config?

**Comment by Knorkebrot** [comment]
Just for the record: EasySIMBL worked for me in 10.10.4 and does still in 10.10.5, you just have to disable and re-enable it after booting, restart Safari and everything's fine again.

**Comment by Gunny123** [comment]
@Knorkebrot Damn, I just went through uninstalling everything and reverting back to original SIMBL. Can you provide screenshots to show it does work?

**Comment by Knorkebrot** [comment]
@Gunny123 http://tmp.kbct.de/proof-no-problem-man.png :ok_hand:

**Comment by Gunny123** [comment]
@Knorkebrot Dope. Thanks.
